packages/react-events/src/Hover.js

```diff
@@ -43,6 +43,14 @@ type HoverEvent = {|
   target: Element | Document,
   type: HoverEventType,
   timeStamp: number,
+  clientX: null | number,
```

Comment:
If we don't need to support existing use of `clientX` (for hover) at FB maybe we shouldn't include it on the event objects

Reply:
True, but I wasn't 100% sure if it wouldn't be used. I'll leave it in for now, we can take a stab at removing a bunch of these later and using higher level versions possibly.